test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs

```diff
@@ -2387,6 +2388,44 @@ public void ToPackageSpec_TwoProjectReferencesToSameProject_DeduplicatesProjectR
         private delegate void TryGetProjectNamesCallback(string projectPath, out ProjectNames projectNames);
         private delegate bool TryGetProjectNamesReturns(string projectPath, out ProjectNames projectNames);
 
+        [Fact]
+        public void ToPackageSpec_NoVersion_ThrowsException()
+        {
```

Comment:
Is this the simplest scenario to validate against? Using the simplest possible scenario will make it easier for others to debug the test and understand what's going on if the test starts to fail.